pydsa/test_cocktail_sort.py

```diff
@@ -0,0 +1,26 @@
+import numpy as np
+
+def sort(s):
```

Comment:
No need to copy + paste the code you want to test. You should import your sort function in the test file to assert various cases. Please check other tests in the repo on how it is done.